distributor-widget/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 17.01.2022
+
+* Introduced new methods on API object `setAdultCount` and `setChildCount` to set initial occupancy.
```

Comment:
I think to make it slightly more clear:
```suggestion
* Introduced new API methods to Distributor instance: [`setAdultCount`](link to reference) and `[setChildCount]`(link to reference).
```
- Instead of object, I'd say Distributor instance. To make it more in line with how we call it in other parts of guide.
- To make it flow a little better. But I'm not English.
- And links to get more details.